index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tabelas Filmes Pixar</title>
+</head>
+<body>
+    <table>
+        <tr>
+            <th>Título do Filme</th>
+            <th>Data de Lançamento</th>
+            <th>Diretor</th>
+            <th>Produtor</th>
+            <th>Orçamento</th>
+            <th>MEU personagem favorito</th>
+        </tr>
+        <tr>
+            <td>Toy Story</td>
+            <td>22 de Novembro de 1995</td>
+            <td>John Lasseter</td>
+            <td>Ralph Guggnheim e Bonnie Arnold</td>
+            <td>30 milhões</td>
+            <td>Rex - Dinossaurin</td>
+        </tr>
+        <tr>
+            <td>Vida de Inseto</td>
+            <td>25 de Novembro de 1998</td>
+            <td>John Lasseter e Andrew Stanton</td>
+            <td>Darla K. Anderson e Keven Reher</td>
+            <td>60 milhões</td>
+            <td>Chucrute - Lagarta</td>
+        </tr>
+        <tr>
+            <td>Monstros S.A.</td>
+            <td>2 de novembro de 2001</td> 
+            <td>Pete Docter</td>
+            <td>Darla K. Anderson</td>
+            <td>115 milhões</td>
+            <td>Sullivan</td>
+        </tr>
+       <tr>
+        <td>Procurando Nemo</td>
+        <td>30 de maio de 2003</td>
+        <td>Andrew Stanton</td>
+        <td>Graham Walters</td>
+        <td>94 milhões</td>
+        <td>Crush - tartaruga maconheira</td>
+       </tr>
```

Comment:
Opa, uma observação que, veremos mais a fundo eventualmente, mas vale se atentar:
A identação é um fator mt importante quando falamos de legibilidade. Lembra que seu código, eventualmente, vai ser revisto tanto por vc, quanto por outros, então é importante que seja fácil de ler e que respeite um padrão.
Dá uma revisada na identação das linhas abaixo
Segue uma sugestão de melhoria:

```suggestion
       <tr>
          <td>Procurando Nemo</td>
          <td>30 de maio de 2003</td>
          <td>Andrew Stanton</td>
          <td>Graham Walters</td>
          <td>94 milhões</td>
          <td>Crush - tartaruga maconheira</td>
       </tr>
```